contracts/consensus/IConsensus.sol

```diff
@@ -18,53 +16,38 @@ import {InputRange} from "../common/InputRange.sol";
 /// For example, a claim may be accepted if it was...
 /// - submitted by an authority or;
 /// - submitted by the majority of a quorum or;
-/// - submitted and not proven wrong after some period of time.
+/// - submitted and not proven wrong after some period of time or;
+/// - submitted and proven correct through an on-chain tournament.
 interface IConsensus {
     /// @notice MUST trigger when a claim is submitted.
     /// @param submitter The submitter address
     /// @param appContract The application contract address
-    /// @param inputRange The input range
-    /// @param epochHash The epoch hash
-    /// @dev Overwrites any previous submissions regarding `submitter`, `appContract` and `inputRange`.
+    /// @param claim The output Merkle root hash
     event ClaimSubmission(
         address indexed submitter,
         address indexed appContract,
-        InputRange inputRange,
-        bytes32 epochHash
+        bytes32 claim
     );
 
     /// @notice MUST trigger when a claim is accepted.
     /// @param appContract The application contract address
-    /// @param inputRange The input range
-    /// @param epochHash The epoch hash
-    /// @dev MUST be triggered after some `ClaimSubmission` event regarding `appContract`, `inputRange` and `epochHash`.
-    /// @dev Overwrites any previous acceptances regarding `appContract` and `inputRange`.
-    event ClaimAcceptance(
-        address indexed appContract,
-        InputRange inputRange,
-        bytes32 epochHash
-    );
+    /// @param claim The output Merkle root hash
+    /// @dev MUST be triggered after some `ClaimSubmission` event regarding `appContract`.
```

Comment:
It's not redundant, because they are talking about different things.

- The first says that the `ClaimAcceptance` event will be triggered every time a claim is accepted.
- The second says that a `ClaimAcceptance` event will always be preceded by a `ClaimSubmission` event about the same `appContract`.

Note that they are orthogonal. You could have:

- The first, but not the second. Example: `ClaimAcceptance(App1, 0xdeadbeef)` is emitted because `0xdeadbeef` was accepted for `App1`, but it was never submitted.
- The second, but not the first. Example: claim `0xdeadbeef` was accepted for `App1` but no event was ever emitted.